ocaml/typing/layouts.ml

```diff
@@ -806,6 +823,15 @@ module Layout = struct
          format_concrete_layout_reason ppf concrete
       | Imported ->
          fprintf ppf "it's imported from another compilation unit"
+      | Imported_type_argument {parent_path; position; arity} ->
+        (match arity, position with
+         | 1, _ -> fprintf ppf "the type argument of %a has this layout"
```

Comment:
Definitely worth making a function to abstract over this pattern. Happily, I think the function can just retun the string between "the" and "type argument". In the case of arity 1, that string will be empty.